server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/visitors/NodeVisitors.java

```diff
@@ -1,5 +1,7 @@
 package org.opentosca.toscana.plugins.cloudfoundry.visitors;
```

Comment:
Shouldn't this class not just be called NodeVisitor instead of NodeVisitors? Since it implements one StrictNodeVisitor, not multiple different ones.

I am NOT talking about the line in question (the package `visitors`) but the actual class name `NodeVisitors` in line 23.

Reply:
done